lib/components/switch/switch.style.dart

```diff
@@ -52,22 +49,25 @@ class SwitchStyles extends StyleRecipe<SwitchStyles> {
 
 Style _outerContainerStyle() => Style(
       box.width(32),
-      box.height(20),
-      padding.all(2),
-      box.borderRadius.all(10),
+      box.height(16),
+      border.all.width(2),
```

Comment:
WIth the latest beta on mix you can do `border.width()` and `border.strokeAlign()` so you don't need to call `all` method

Reply:
perfect